reference-adapter/pspcl-ifix-adapter/src/main/java/org/egov/entity/PspclBillDetail.java

```diff
@@ -0,0 +1,38 @@
+package org.egov.entity;
+
+import com.vladmihalcea.hibernate.type.json.JsonBinaryType;
+import lombok.Data;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@Entity
+@Table(name = "pspcl_bill_detail")
+@TypeDef(name = "jsonb", typeClass = JsonBinaryType.class)
+public class PspclBillDetail {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", unique = true, nullable = false)
+    private Long id;
+
+    private String ORDERBYCOLUMN;
```

Comment:
Can we rename this so that other references of the same throughout the project have a sensible name and don't create confusion?

Reply:
To keep in sync with pspcl bill results' attribute..we have kept this name.